sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/Aggregation.java

```diff
@@ -24,43 +28,76 @@
   private static final ThrottlingLogger logger =
       new ThrottlingLogger(Logger.getLogger(Aggregation.class.getName()));
 
-  private static final Aggregation NONE = Aggregation.make("none", unused -> null);
+  private static final Aggregation NONE = new NoAggregation();
   private static final Aggregation DEFAULT =
```

Comment:
what do you say to pulling this out into a separate (inner) class like the `NoAggregation` above?

Reply:
done!